valhalla/meili/state.h

```diff
@@ -123,6 +123,14 @@ class StateContainer {
     return static_cast<StateId::Time>(columns_.size());
   }
 
+  // Check to see if we have the minimum number of measurements and edge candidates to perform a map
+  // match. We need at least two measurements with a non-zero number of edge candidates.
+  bool HasMinimumCandidates() {
+    size_t num_nonempty =
+        std::count_if(columns_.begin(), columns_.end(), [](Column& col) { return !col.empty(); });
+    return size() > 1 && num_nonempty > 1;
```

Comment:
@kevinkreiser let me know if the mapping of these thresholds to 443 makes sense.

Reply:
i think the `size` check is superfluous. a request wont get past the parsing phase if it has no input candidates but it doesnt hurt anything. i was t hinking that `> 0` was good enough but i dont feel strong about it.